code/modules/projectiles/guns/manufacturer/frontier_import/ballistics.dm

```diff
@@ -0,0 +1,229 @@
+/obj/item/gun/ballistic/automatic/pistol/mauler
+	name = "Mauler machine pistol"
+	desc = "A full auto machine pistol originating from the Shoal. Insane stopping power, but mostly useless outside of CQC with unarmored targets. Chambered in 9mm."
+	icon = 'icons/obj/guns/manufacturer/frontier_import/48x32.dmi'
+	lefthand_file = 'icons/obj/guns/manufacturer/hunterspride/lefthand.dmi'
+	righthand_file = 'icons/obj/guns/manufacturer/hunterspride/righthand.dmi'
+	mob_overlay_icon = 'icons/obj/guns/manufacturer/frontier_import/onmob.dmi'
+
+	icon_state = "mauler"
+	item_state = "hp_generic"
+	w_class = WEIGHT_CLASS_NORMAL
+	mag_type = /obj/item/ammo_box/magazine/m9mm_mauler
+	fire_delay = 0.06 SECONDS
+
+	gun_firemodes = list(FIREMODE_FULLAUTO)
+	default_firemode = FIREMODE_FULLAUTO
+
+	spread = 25
+	spread_unwielded = 50
+	recoil = 1
+	recoil_unwielded = 4
+	fire_sound = 'sound/weapons/gun/pistol/mauler.ogg'
+
+	rack_sound = 'sound/weapons/gun/pistol/candor_cocked.ogg'
+
+	lock_back_sound = 'sound/weapons/gun/pistol/slide_lock.ogg'
+	bolt_drop_sound = 'sound/weapons/gun/pistol/candor_cocked.ogg'
+
+	load_sound = 'sound/weapons/gun/pistol/candor_reload.ogg'
+	load_empty_sound = 'sound/weapons/gun/pistol/candor_reload.ogg'
+	eject_sound = 'sound/weapons/gun/pistol/candor_unload.ogg'
+	eject_empty_sound = 'sound/weapons/gun/pistol/candor_unload.ogg'
+
+/obj/item/gun/ballistic/automatic/pistol/mauler/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/automatic_fire, 0.06 SECONDS)
+
+/obj/item/ammo_box/magazine/m9mm_mauler
+	name = "mauler machine pistol magazine (9mm)"
+	desc = "A long, 12-round magazine designed for the Mauler machine pistol. These rounds do okay damage, but struggle against armor."
+	icon_state = "mauler_mag-1"
+	base_icon_state = "mauler_mag"
+	ammo_type = /obj/item/ammo_casing/c9mm
+	caliber = "9mm"
+	max_ammo = 12
+
+/obj/item/ammo_box/magazine/m9mm_mauler/update_icon_state()
+	. = ..()
+	icon_state = "[base_icon_state]-[!!ammo_count()]"
+
+/obj/item/gun/ballistic/automatic/pistol/spitter
+	name = "\improper Spitter"
+	desc = "An Old Frontiersmen machine pistol. While its origin is controversial, and it is primarily associated with criminals, it is widely cloned and used in the frontier. Chambered in 9mm."
```

Comment:
"An open-bolt submachine gun favored by the Frontiersmen. This design's origins are unclear, but its simple, robust design has been widely copied throughout the Frontier, and it is stereotypically used by pirates and various criminal groups that value low price and ease of concealment. Chambered in 9mm."

Referring to it as a submachine gun here to avoid confusion with the Mauler, and because it better suits the Spitter's combat role. The other machine pistols in the game are generally inaccurate self-defense weapons with high rates of fire and small magazines.